recipe/meta.yaml

```
@@ -1,42 +1,49 @@
 {% set name = "zope.sqlalchemy" %}
-{% set version = "1.1" %}
+{% set version = "2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 81554c5b03fbf924c4144ef835b7900271fbd85cfe81cb6bd95e3ab7aa85189f
+  sha256: cdf70cd57b8beb0ca9a81754abbf5c80ef0b53e8d8b2d894ac20bfc73870df12
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
+  # s390x missing transaction
+  skip: true  # [s390x or py<37]
 
 requirements:
   host:
     - python
     - setuptools
+    - pip
+    - wheel
   run:
     - python
-    - sqlalchemy >=0.5.1
+    - sqlalchemy >=1.1,<2
```

```suggestion
    - sqlalchemy >=1.1,!=1.4.0,!=1.4.1,!=1.4.2,!=1.4.3,!=1.4.4,!=1.4.5,!=1.4.6,<2
```
It's a long line, but that can be kept.
https://github.com/zopefoundation/zope.sqlalchemy/blob/2.0/setup.py#L50